server/src/main/java/io/crate/planner/CreateUserMappingPlan.java

```diff
@@ -55,6 +65,23 @@ public void executeOrFail(DependencyCarrier dependencies,
                               RowConsumer consumer,
                               Row params,
                               SubQueryResults subQueryResults) throws Exception {
+        Metadata metadata = dependencies.clusterService().state().metadata();
```

Comment:
The `plannerContext` contains also a clusterState that should be preferred - to ensure all planner decisions are based on the same state, and it also makes testing a bit easier, as the dependencies/clusterService don't have to be mocked.